neverwinter/nwscript/native/scriptcompfinalcode.cpp

```diff
@@ -93,6 +93,42 @@ int32_t CScriptCompiler::GenerateFinalCodeFromParseTree(CExoString sFileName)
 		OutputWalkTreeError(nReturnValue, NULL);
 	}
 
+	if (!m_sGraphvizPath.IsEmpty())
+	{
+		CExoString sDotFile = m_sGraphvizPath + "/" + sFileName + ".dot";
+		FILE *f = fopen(sDotFile.CStr(), "w");
+		if (f)
```

Comment:
What if this is falsey? Not really a good way to handle it except abort, I guess.

Reply:
I didn't want to fail the compile. I was gonna emit a message and move on, but there's no good way to emit messages, other than `fprintf(stderr, ...)` which didn't seem appropriate.